### DISCUSSION TITLE
Iris Webinterface not working, no playback

### DISCUSSION THREAD

**Discussion by theghostronaut** [MiczFlor/RPi-Jukebox-RFID#1490] [Q&A]
My phoniebox setup with Spotify worked fine, but took a long time too boot. So I followed the tipps by Splittscheid (https://splittscheid.de/phoniebox-schneller-booten-fehler-beheben/#ftoc-musikdatenbank-aktualisieren) to speed up booting the Spotify version. I basically only disabled bluetooth AND changed the playlist loading. After that, my IRIS web interface doesn't load anymore and I'm not getting any audio neither from local nor spotify files.

One thing I did was, the last time I was still able to access the web interface, there was a "upgrade to new version" button in the settings I clicked. After that, the issues accord (it didn't show me any updates being made). Not sure if this had anything to do with it. According to mopidy status, the service is running fine. Any idea what is going on?

**Comment by s-martin** [comment]
Did it work after you reverted your changes?

pinging @splitti
